browsers/oculus.json

```diff
@@ -0,0 +1,218 @@
+{
+  "browsers": {
+    "oculus": {
+      "accepts_flags": true,
```

Comment:
Same as comment above, just wanna double check does the oculus browser support flags?

Reply:
Yes! Oculus Browser supports flags by entering `chrome://flags` in the address bar. We don't navigate links to that URL for security reasons but entering it directly brings up a page to search and set flags similar to Chrome for Android.